symfony/plugins/orangehrmPimPlugin/Dao/ReportingMethodConfigurationDao.php

```diff
@@ -160,4 +161,21 @@ public function isExistingReportingMethodName(string $reportingMethodName): bool
             throw new DaoException($e->getMessage(), $e->getCode(), $e);
         }
     }
+
+    /**
+     * @return array
+     * @throws DaoException
+     */
+    public function reasonsInUse(): array
```

Comment:
refactor method name, it's nor reasons in reporting method,
And refactor all other variable names

Reply:
sure used reportingMethodsInUse